src/adapter.c

```diff
@@ -437,13 +437,13 @@ int close_adapter(int na)
 	}
 
 	LOG("closing adapter %d  -> fe:%d dvr:%d, sock:%d, fe_sock:%d", na, ad->fe, ad->dvr, ad->sock, ad->fe_sock);
-	ad->enabled = 0;
-	if (ad->close)
-		ad->close(ad);
 	//close all streams attached to this adapter
 	//	close_streams_for_adapter (na, -1);
 	mark_pids_deleted(na, -1, NULL);
-	update_pids(na);
+	// update_pids(na); // No update pids as the adapter is closing!
```

Comment:
If you update the pids then the a new PLAY command is sended... but we're sending a TEARDOWN, so the commit is not required: we only need to free the pid list.